src/core/gps/qgsgpsconnection.h

```diff
@@ -88,14 +88,20 @@ class CORE_EXPORT QgsSatelliteInfo
      * Signal strength (0-99dB), or -1 if not available.
      */
     int signal = -1;
+    /**
```

Comment:
```suggestion
    int signal = -1;

    /**
```

Oops... it was my fault...

Reply:
@agiudiceandrea ok